java_src/foam/dao/index/AltIndex.java

```diff
@@ -1,96 +0,0 @@
-/**
```

Comment:
Just noticed that they don't have the _generated-code_ header.

So I assume `src/foam/dao/index/AltIndex.js` does not generate a `build/foam/dao/index/AltIndex.java` file?